Documentation/AndroidInstallation.md

```diff
@@ -66,21 +66,23 @@ compileOptions {
 }
 ```
 
-Replace to:
-
-```gradle
-    compileSdkVersion 27
-    buildToolsVersion '27.0.3'
-```
-
-Also inside of `dependencies` replace all the `compile` by `implementation` otherwise you'll get deprecation warnings.
+Inside of `dependencies` replace all the `compile` by `implementation` otherwise you'll get deprecation warnings.
```

Comment:
This is not needed (yet) and has really nothing to do with how this plugin functions. Can you please drop it?

Reply:
Sure!